.github/workflows/on_push.yml

```diff
@@ -0,0 +1,57 @@
+name: CI
+
+on:
+  push:
+    branches:
+    - master
+    paths:
+    - 'Dockerfile'
+    - '.github/workflows/on_push.yml'
+
+jobs:
+  build-publish:
+    name: Build & Publish Docker Image
+    runs-on: ubuntu-latest
+    env:
+      IMAGE_NAME: docker.pkg.github.com/${{ github.repository }}/decidim-generator
+      TAG: ${{ github.sha }}
+
+    steps:
+    - name: Fetch and Set Ruby Version
+      id: ruby-version
+      run: |
+        echo ::set-output name=version::$(curl -s https://raw.githubusercontent.com/decidim/decidim/develop/.ruby-version)
```

Comment:
why doesn't it check the current branch for the ruby version? I'm thinking about what would happen when upgrading it.

Reply:
yeah, I wanted to get some feedback on this. At the time the PR was posted the latest tag was v0.22.0, and ruby was at 2.5.3 there. There's a bug related to this outdated ruby version, and so I worried about publishing an image using it.

It's definitely not ideal, it should really fetch the version from the latest tag. I'll add it in

Reply:
I'd rather stick to the current branch's ruby version and solve the issue in the 0.22 branch instead. In the end, this workflow is not responsible for that.

Reply:
I agree with @sauloperez that we should get the Ruby version from the current branch otherwise we should have inconsistencies and incompatibilities

Reply:
Good points - agree as well. I'm working on making it fetch from the latest tag then.

Reply:
done on https://github.com/decidim/docker/pull/50/commits/069f74f425370124b17222226444509abe3985d5